pkg/cmd/server/start/start_master.go

```diff
@@ -533,6 +534,16 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 	oc.RunServiceAccountPullSecretsControllers()
 	oc.RunSecurityAllocationController()
 
+	// TODO(miminar): run origin resource quota controller by default after 3.2
+	// run origin resourcequota controller only if the quota admission plugin is enabled
+	runOriginResourceQuotaController := false
```

Comment:
Running it makes sense only if the admission plugin is enabled. In the future, there may be other admission plugins dependent on it as it will manage more resources.

On the other hand, there's no harm letting it run with the admission p. disabled other than wasted cpu cycles.

Would you rather have it configured with a flag?